src/wwwroot/template-editor/adapterService.js

```diff
@@ -8,10 +8,11 @@
 
   adapterService.$inject = [
     '$http',
-    '$q'
+    '$q',
+    '$window'
   ];
 
-  function adapterService($http, $q) {
+  function adapterService($http, $q, $window) {
```

Comment:
This is something totally opinable, so, feel free to take my advice or not.

Did you see that there is a line for each dependency in _adapterService.$inject_? I would link to use the same approach for parameters in this constructor:

```js
adapterService.$inject = [
  '$http',
  '$q'
  '$window'
];
 
function adapterService(
  $http, 
  $q, 
  $window) {
```

In my opinion, it is easier to read (when start to be a lot of them) and also it is friendly with the diffs.